.github/CODEOWNERS

```diff
@@ -1441,6 +1443,7 @@ spec/models/message_spec.rb @department-of-veterans-affairs/vfs-mhv-secure-messa
 spec/models/mhv_opt_in_flag_spec.rb @department-of-veterans-affairs/vfs-mhv-secure-messaging @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 spec/models/mpi_data_spec.rb @department-of-veterans-affairs/octo-identity
 spec/models/onsite_notification_spec.rb @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+spec/models/pega_table_spec.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

Comment:
You also need to add your github team as reviewers. Which team will be responsible for these files? If you don't have a team, pleas create one. https://github.com/orgs/department-of-veterans-affairs/teams

Reply:
@rmtolmach  I see @department-of-veterans-affairs/champva-engineering in our github teams but in CODEOWNERS, its  not recognized

Reply:
@rmtolmach  Can you help out?

Reply:
Yes! I've added champva-engineering to the vets-api repo as collaborators with write access so the warnings are gone 👍